### DISCUSSION TITLE
Python API Advice

### DISCUSSION THREAD

**Discussion by maybeRXT** [community/community#138272] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

Hello! I am just getting started a data scientist in Python and I was wondering if anyone has any cool or interesting API's or Imports they would like to share with me for some simple starter projects. I am a bit rusty in python as I have not done it for the better part of a year But I will be sure to study up. Thank you.

**Comment by swapnilsparsh** [comment]
You can try out this -> https://github.com/swapnilsparsh/YoutubeDownloader

**Reply by maybeRXT** [reply]
Thank you! Have a good day sir
